cypress/s3_monitor.py

```diff
@@ -0,0 +1,184 @@
+import boto3
+import json
+import quopri
+import time
+import re
+import argparse
+from pathlib import Path
+
+
+class S3Monitor:
+
+    MAILBOX_BUCKET = "opg-lpa-casper-mailbox"
+    ACTIVATION_EMAILS_PATH = (Path(__file__).parent / "activation_emails").resolve()
+
+    def __init__(self, options):
+        """
+        :param options: dict with v (verbose) and c (is in CI) boolean properties
+        """
+        self.verbose = options["v"]
+        self.in_ci = options["c"]
+        self.s3Client = self.assume_role_and_get_client()
+
+    def assume_role_and_get_client(self):
+        sts = boto3.client(
+            "sts",
+            region_name="eu-west-1",
```

Comment:
make this a parameter, will help with DR

Reply:
What's the correct value to use from terraform for this? Presumably I can pass an env var over which contains this.